src/runners/cairoRunner.ts

```diff
@@ -30,14 +31,23 @@ export class CairoRunner {
     this.program = program;
     const mainId = program.identifiers.get('__main__.main');
     const mainOffset = mainId !== undefined ? mainId.pc ?? 0 : 0;
+    const builtins = program.builtins;
 
     this.vm = new VirtualMachine();
     this.programBase = this.vm.memory.addSegment();
     this.executionBase = this.vm.memory.addSegment();
 
+    const builtin_stack = builtins.map((builtin) => {
+      switch (builtin) {
+        case 'bitwise':
+          return this.vm.memory.addSegment(new Bitwise());
+        default:
```

Comment:
I'd rather export an object/namespace/class from builtins so that here we can do only `program.builtins.map((b) => new Builtin(b)).map((b) => b.initial_stack()).flat()`

Reply:
especially here at the cairoRunner level, I don't really want to have this detailed list of what builtin name corresponds to what builtin class, but I'd rather export an object so that the module builtin is self-sufficient